### ISSUE TITLE
Internal links

### ISSUE THREAD

**Issue by countofsanfrancisco** [paramt/url-checker#2]
Hi,

Does this tool also check internal links? For example, if I do something like this in markdown:

```
[a link to somewhere on same page](#somewhere_link)
```

Do you check to make sure the `somewhere_link` is anchored somewhere on the same page?

**Comment by paramt** [comment]
No, right now the tool only checks for external links. I'm not planning on updating it anytime soon, but I'll keep this issue open if anyone's interested in adding this feature.
